receptor_affinity/mesh.py

```diff
@@ -136,16 +137,11 @@ def get_routes(self):
         else:
             return read_and_parse_metrics(routes)
 
-    def validate_routes(self):
+    def validate_routes(self) -> None:
         if not self.active:
-            raise Exception("Can't get routes from a stopped node")
-        print(f"****====TRYING COMPARE {self.name}")
-        node_routes = self.get_routes()
-        control_routes = self.mesh.generate_routes()
-        if node_routes and control_routes:
-            return self.mesh.compare_routes(node_routes, control_routes)
-        else:
-            return False
+            raise RouteUnavailableError(self)
```

Comment:
Route or Node? I'm wondering as we get further into this we are going to have Node that are unqueriable, so perhaps it should be a more generic `NodeUnavailable`, or `NodeFunctionUnavailable`? What do you think?

Reply:
I do like the idea of a `NodeInactiveError` or somesuch. It more accurately describes the underlying issue.